### PR TITLE
Add ignore_case argument to element_should_contain 

### PR DESCRIPTION
**Pull request by SidePlace**

### CODE REVIEW

**Comment by SidePlace** [comment]
Trying to help on #849 
Forgot to mention bug ID in the commit message

**Comment by aaltat** [comment]
Code looks OK, but PR is missing test. PR are not merged without test backing them up.

**Comment by SidePlace** [comment]
Hi @aaltat I just pushed my tests for it. Let me know if I'm doing something improperly.

**Comment by aaltat** [comment]
We have had lately problems with the Travis CI, sometimes the test execution stops. It seems that this PR is also suffering from the same problem. I did restart the two failed runs and lets see how it goes.

File: test/acceptance/ignoreCase.robot [hunk 1]
```diff
@@ -0,0 +1,19 @@
+*** Settings ***
```
Comment (aaltat) [inline test/acceptance/ignoreCase.robot:1]: There already is a existing suite for this keyword, in ./test/acceptance/keywords/content_assertions.robot You have two options:
1) Move tests from here to the content_assertions suite
2) Move the `Element Should Contain` keyword test all in same and new suite. Also change the suite name to describe the whole keyword, not just the new feature.

File: test/acceptance/ignoreCase.robot [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Library           SeleniumLibrary
+Test Teardown     Close All Browsers
+
+*** Variables ***
+${ISSUE_URL}      https://github.com/robotframework/SeleniumLibrary/issues/849
```
Comment (aaltat) [inline test/acceptance/ignoreCase.robot:6]: Using links to external resources are not allowed. The acceptance test runner starts small http server. which must be used to serve the required html files. Please take an example from the ./test/acceptance/keywords/content_assertions.robot suite. 

The test logic is OK,

**Review by aaltat — CHANGES_REQUESTED** [review]
There are two things to fix.

**Comment by aaltat** [comment]
If you want credit for your work, please add your name and short description what you have done in the https://github.com/robotframework/SeleniumLibrary/blob/master/CHANGES.rst

**Comment by aaltat** [comment]
Closing on favour of #1043.

**Pull request closed without merging**